### ISSUE TITLE
Trouble with 4k60 videos

### ISSUE THREAD

**Issue by Brancliff** [bbilly1/tubearchivist#41]
I tried running some 4k 60FPS videos through Tubearchivist to see if they would download. They were able to go through, but the sound was missing.

Since it uses yt-dlp to download the videos, I tried running the videos through that to see if that was it. But the videos seemed to go through fine. Maybe it has something to do with the filetype? YouTube downloads above 1080p are sent out in .mkv, but Tubearchivist was able to convert it to .mp4.

Here are the videos I've used:

https://www.youtube.com/watch?v=LXb3EKWsInQ
https://www.youtube.com/watch?v=mkggXE5e2yk

**Comment by bbilly1** [comment]
So I tried both of your links, they both downloaded fine, video and sound, both is fine. I tried with disabling the format by setting it to `0` and by setting it to `bestvideo+bestaudio`, the result was the same 4K, and 60fps and working sound. Also the media file played fine both over the web interface and also by opening the file in vlc or mpv.  

What was the format setting you used?
What's the output of `mediainfo` on the media file?

Could also be some other hardware based limitations, video player limitations, graphics driver, etc...
